packages/react-native-renderer/src/ReactFabricHostConfig.js

```diff
@@ -137,17 +137,23 @@ class ReactFabricHostComponent {
   }
 
   measure(callback: MeasureOnSuccessCallback) {
-    fabricMeasure(
-      this._internalInstanceHandle.stateNode.node,
-      mountSafeCallback_NOT_REALLY_SAFE(this, callback),
-    );
+    const {stateNode} = this._internalInstanceHandle;
+    if (stateNode != null) {
```

Comment:
Should this be silently ignored? Does it indicate a bug somewhere? It seems a lot like the warning we show when you call setState on an unmounted component. What do you think about a warning like "you called a command on an unmounted component"?

Reply:
I feel like I should mirror what React does when you update state on an unmounted component. Should I be using `console.warn` or `console.error`? And should it only be done in `__DEV__`?